make_filters/make_filters.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+
+import data_storage
+import json
+import yaml
+import os
+from cloudferrylib.utils import utils as utl
+LOG = utl.get_log(__name__)
+
+
+MIGRATE_VM_PREFIX = 'migrate_vm_'
+
+
+def delete_relations():
+    LOG.info("started deleting old VM to hypervisor relations")
+    keys = data_storage.keys(MIGRATE_VM_PREFIX + '*')
+    for key in keys:
+        data_storage.delete(key)
+    LOG.info("Relation deleting done. %s records was removed." % len(keys))
+
+
+def check_filter_folder(filter_folder):
+    if not os.path.exists(filter_folder):
+        os.makedirs(filter_folder)
+
+
+def make(filter_folder, images_date):
+    delete_relations()
```

Comment:
Do we really need to store the data in Redis, if it's being removed every time? Maybe it would be easier to simply convert "condensation" data in python process, without storing intermediate results in Redis?